components/src/main/res/values/colors.xml

```diff
@@ -32,7 +32,7 @@
     <color name="hmrc_grey_2">#FFB1B4B6</color>
     <color name="hmrc_grey_3">#FFF3F2F1</color>
     <color name="hmrc_pink">#FFCA2B75</color>
-    <color name="hmrc_yellow">#FFFFDD00</color>
+    <color name="hmrc_yellow">#FFBF47</color>
```

Comment:
think you missed out two additional "FF" at the beginning, not sure what/why we need it, but all other ones have it.

Reply:
Addressed in 37cfbf34f3f4f1def3882f38baa1ef74fbf5a613